server/src/main/java/io/unitycatalog/server/service/TemporaryModelVersionCredentialsService.java

```diff
@@ -29,7 +36,19 @@ public TemporaryModelVersionCredentialsService(CredentialOperations credentialOp
     }
 
     @Post("")
+    @AuthorizeExpression("""
```

Comment:
So to match how DB UC handles this, we need to apply the FunctionExecute auth action when the requested credential is READ, and apply FunctionUpdate auth action when the requested credential is a READ/WRITE.

Reply:
See: https://src.dev.databricks.com/databricks/universe@ae2dac8a44b0e98d8ff18042bcf4a626476e36ad/-/blob/managed-catalog/src/main/scala/com/databricks/backend/managedcatalog/rpc/core/credential/GenerateTemporaryModelVersionCredentialHandler.scala?L252-256